lib/utils/fhlogin.js

```diff
@@ -63,6 +64,6 @@ function done(cb) {
     log.silly(response, "login response");
     log.error(JSON.stringify(data), "login");
     return cb(new Error((response && response.statusCode) + " " +
-      "Could not login! " + '\n' + JSON.stringify(data)));
+			i18n._("Could not login! ") + '\n' + JSON.stringify(data)));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace all tab characters in this file by sequences of white-spaces. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATabCharacter)